table_reconstruction/utils/cell_utils.py

```diff
@@ -0,0 +1,262 @@
+from typing import List, Tuple
+from shapely.geometry import LineString
+from table_reconstruction.utils.lines_utils import is_line
```

Comment:
anh nghĩ nên để import thế này
```suggestion
from .lines_utils import is_line
```

Reply:
Em thấy hợp lý hơn đấy :v